test_collections/sdk_tests/support/chip_tool/chip_tool.py

```diff
@@ -98,7 +98,7 @@
 DOCKER_CREDENTIALS_DEVELOPMENT_PATH = "/credentials/development"
 
 # Websocket runner
-BACKEND_ROOT = Path(__file__).parents[2]
+BACKEND_ROOT = Path(__file__).parents[4]
```

Comment:
If we don't need backend root really, we could stop closer to what we need. Eg a directly for `SDK_TESTS_PATH =  Path(__file__).parents[2]`